doc/protocol.md

```diff
@@ -289,9 +289,10 @@ is no longer supported.
 Additional "open" command options are needed to open a channel of this
 type:
 
- * "bus": The DBus bus to connect to either "session" or "system",
-   defaults to "system" if not present. If set to "internal" then this
-   channel will communicate with the internal bridge DBus connection.
+ * "bus": The DBus bus to connect to either "session", "system" or a dbus
+   supported address. Defaults to "system" if not present. If set to
```

Comment:
I was trying to avoid adding a new param, but if the name is important then we can add an address parameter that is looked at if bus is null. We'll need a matching capability. Do you prefer that?